sdk/core/core-amqp/src/cbs.ts

```diff
@@ -234,11 +235,11 @@ export class CbsClient {
       });
       logger.verbose("[%s] The CBS response is: %O", this.connection.id, responseMessage);
       return this._fromRheaMessageResponse(responseMessage);
-    } catch (err: any) {
+    } catch (err) {
       logger.warning(
         "[%s] An error occurred while negotiating the cbs claim: %s",
         this.connection.id,
-        `${err?.name}: ${err?.message}`
+        isError(err) ? `${err?.name}: ${err?.message}` : String(err)
```

Comment:
optional chain shouldn't be required if the type is guarded by isError

```suggestion
        isError(err) ? `${err.name}: ${err.message}` : String(err)
```

Reply:
Updated